packages/lesswrong/components/users/UserTooltip.tsx

```diff
@@ -11,7 +11,7 @@ const styles = (theme: ThemeType) => ({
       border: `1px solid ${theme.palette.grey[120]}`,
       boxShadow: theme.palette.boxShadow.eaCard,
       padding: 12,
-      marginTop: 2,
+      top: 2,
```

Comment:
Unrelated to the rest of this PR, but I was getting a console warning because of this ("margin can't be applied due to being anchored to a parent element" or something to that effect)